lib/App/index.js

```diff
@@ -311,6 +308,29 @@ class App {
 		this.debug(`Validated successfully`);
 		
 	}
+
+    _checkZwaveForSetting(driver, setting) {
+		if (!setting.zwave) return;
+        else if (!setting.zwave.index || !setting.zwave.size) throw new Error(`Missing property in "zwave" at ${driver.id}, ${setting.id}`);
```

Comment:
This will throw a weird error if `zwave.index === 0` or `zwave.size === 0` (which should not happen of course).

Reply:
Will fix